<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>530</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>530</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>20</y>
      <width>871</width>
      <height>511</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Arial</family>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(170, 170, 255);</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <property name="font">
      <font>
       <family>Arial</family>
       <pointsize>9</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <attribute name="title">
      <string>Tab 1</string>
     </attribute>
     <widget class="QWidget" name="widget" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>800</width>
        <height>500</height>
       </rect>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>500</width>
        <height>500</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>800</width>
        <height>500</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(170, 170, 255);</string>
      </property>
      <widget class="QLabel" name="labelColorVerde">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>260</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>VERDE:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="id">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>50</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelCOLOR">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>300</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>COLOR:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelColorRojo">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ROJO:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="destinoY">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>210</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="rojo">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>300</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelVELOCIDAD">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>10</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>VEL.:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelORIGEN_X">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>90</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ORIGEN: X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="velocidad">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>50</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="destinoX">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>130</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelDESTINO_X">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>90</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DESTINO: X</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelID">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>10</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ID:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelColorAzul">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>260</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AZUL:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="labelDESTINO_Y">
       <property name="geometry">
        <rect>
         <x>240</x>
         <y>170</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DESTINO: Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="verde">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>300</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="origenY">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>210</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="azul">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>300</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="origenX">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>130</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="cursor">
        <cursorShape>PointingHandCursor</cursorShape>
       </property>
       <property name="wrapping">
        <bool>false</bool>
       </property>
       <property name="maximum">
        <number>500</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelORIGEN_Y">
       <property name="geometry">
        <rect>
         <x>120</x>
         <y>170</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>ORIGEN: Y</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="abInicio">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>430</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AGREGAR INICIO</string>
       </property>
      </widget>
      <widget class="QDoubleSpinBox" name="distancia">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>370</y>
         <width>90</width>
         <height>30</height>
        </rect>
       </property>
       <property name="decimals">
        <number>2</number>
       </property>
      </widget>
      <widget class="QLabel" name="labelDistancia">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>370</y>
         <width>95</width>
         <height>35</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>DISTANCIA</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="abMostrar">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>430</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>MOSTRAR</string>
       </property>
      </widget>
      <widget class="QPushButton" name="abFinal">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>430</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>AGREGAR FINAL</string>
       </property>
      </widget>
      <widget class="QGraphicsView" name="graphicsView">
       <property name="geometry">
        <rect>
         <x>415</x>
         <y>20</y>
         <width>371</width>
         <height>241</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="abLimpiar">
       <property name="geometry">
        <rect>
         <x>610</x>
         <y>430</y>
         <width>150</width>
         <height>40</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial</family>
         <pointsize>12</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>LIMPIAR</string>
       </property>
      </widget>
      <widget class="QPlainTextEdit" name="plainTextEdit">
       <property name="geometry">
        <rect>
         <x>415</x>
         <y>270</y>
         <width>371</width>
         <height>151</height>
        </rect>
       </property>
      </widget>
      <widget class="QGroupBox" name="groupBox">
       <property name="geometry">
        <rect>
         <x>210</x>
         <y>339</y>
         <width>181</width>
         <height>81</height>
        </rect>
       </property>
       <property name="title">
        <string>Ordenamiento</string>
       </property>
       <widget class="QRadioButton" name="rBId">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>id</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBDistancia">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Distancia</string>
        </property>
       </widget>
       <widget class="QRadioButton" name="rBVelocidad">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>82</width>
          <height>17</height>
         </rect>
        </property>
        <property name="text">
         <string>Velocidad</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab2">
     <attribute name="title">
      <string>Tab 2</string>
     </attribute>
     <widget class="QTableWidget" name="tableWidget">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>10</y>
        <width>681</width>
        <height>391</height>
       </rect>
      </property>
      <column>
       <property name="text">
        <string>Id</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Origen</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Destino</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Velocidad</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Distancia</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Color</string>
       </property>
      </column>
     </widget>
     <widget class="QPushButton" name="abBuscar">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>420</y>
        <width>100</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Busca Partículas</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineaBuscar">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>420</y>
        <width>200</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
       </font>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
      <property name="dragEnabled">
       <bool>false</bool>
      </property>
      <property name="readOnly">
       <bool>false</bool>
      </property>
      <property name="placeholderText">
       <string>Introduce el Identificador</string>
      </property>
     </widget>
     <widget class="QPushButton" name="abMostrarParticulas">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>420</y>
        <width>171</width>
        <height>40</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>9</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Mostrar Partículas (Todas)</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QMenuBar" name="menubar">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>800</width>
      <height>21</height>
     </rect>
    </property>
    <widget class="QMenu" name="menuArchivo">
     <property name="title">
      <string>Archivo</string>
     </property>
     <addaction name="actionAbrir_"/>
     <addaction name="actionGuardar_"/>
    </widget>
    <addaction name="menuArchivo"/>
   </widget>
  </widget>
  <action name="actionAbrir">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
  <action name="actionAbrir_">
   <property name="text">
    <string>Abrir</string>
   </property>
  </action>
  <action name="actionGuardar_">
   <property name="text">
    <string>Guardar</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
